pkg/grafana/grafana.go

```diff
@@ -104,29 +112,21 @@ func doRequest(c *http.Client, req *http.Request) error {
 	return nil
 }
 
-type Interface interface {
-	Dashboards() DashboardsInterface
-}
-
 type Clientset struct {
 	BaseUrl    *url.URL
 	HTTPClient *http.Client
 }
 
-func New(baseUrl *url.URL) *DashboardsClient {
-	return &DashboardsClient{
+func New(baseUrl *url.URL) *APIClient {
+	return &APIClient{
 		BaseUrl:    baseUrl,
 		HTTPClient: http.DefaultClient,
 	}
 }
 
-func (c *Clientset) Dashboards() DashboardsInterface {
-	return NewDashboardsClient(c.BaseUrl, c.HTTPClient)
-}
-
-func makeUrl(baseUrl *url.URL, endpoint string) string {
+func makeUrl(baseURL *url.URL, endpoint string) string {
 	result := &url.URL{}
-	*result = *baseUrl
+	*result = *baseURL
```

Comment:
you don't need line 128, do you?

Reply:
Good spot - it is not needed. Added a unit test to verify as well.